README.md

```diff
@@ -148,7 +145,7 @@ Circuit Maintenance Notification #0
 ```
 
 ```bash
-$ circuit-maintenance-parser --raw-file tests/integration/data/zayo/zayo1.html --parser zayo
+$ circuit-maintenance-parser --data-file tests/unit/data/zayo/zayo1.html --data-type html --provider-type zayo
```

Comment:
Just a friendly reminder that before we merge this, we probably want to also include a set of CHANGELOG updates spelling out what's changed between 1.x and 2.x, both in terms of CLI usage and integration as a library (e.g. into nautobot-circuit-maintenance).